trello.py

```diff
@@ -0,0 +1,47 @@
+import os
+import requests
+
+host = "https://api.trello.com/1/"
+
+
+def get_trello_board_id():
+    board = get_trello("members/me/boards?")
+    return board[0]["id"]
+
+
+def get_trello(url):
+    url = add_trello_host_token_and_key(url)
+    response = requests.get(url=url).json()
+    return response
```

Comment:
Minor point, but you could just do this on one line, `return requests.get(...`
Same with the other functions below